src/test/kotlin/com/linchpino/core/controller/JobPositionControllerTestIT.kt

```diff
@@ -88,16 +97,44 @@ class JobPositionControllerTestIT{
 			.andExpect(jsonPath("$.content[4].title").value("Marketing Specialist"))
 	}
 
+
+	@Test
+	fun `test get interview types by job id returns interview types for that job id`() {
+		val all = jobPositionRepository.findAll()
+		val jobId = all.first { it.title == "Software Engineer" }.id
+		// Perform the GET request to the endpoint
+		mockMvc.perform(get("/api/jobposition/$jobId/interviewtype"))
+			.andExpect(status().isOk)
+			.andExpect(jsonPath("$").isArray())
+			.andExpect(jsonPath("$.length()").value(2))
+			.andExpect(jsonPath("$[0].id").isNumber)
+			.andExpect(jsonPath("$[0].title").value("InterViewTyp_1"))
+			.andExpect(jsonPath("$[1].id").isNumber)
+			.andExpect(jsonPath("$[1].title").value("InterViewTyp_2"));
+	}
+
+	@Test
+	fun `test get interview types by job id only returns interview types belonging to that id`() {
+		val jobId = jobPositionRepository.findAll().first { it.title == "Data Scientist" }.id
+		// Perform the GET request to the endpoint
+		mockMvc.perform(get("/api/jobposition/$jobId/interviewtype"))
+			.andExpect(status().isOk)
+			.andExpect(jsonPath("$").isArray())
+			.andExpect(jsonPath("$.length()").value(1))
+			.andExpect(jsonPath("$[0].id").isNumber)
+			.andExpect(jsonPath("$[0].title").value("InterViewTyp_2"));
+	}
```

Comment:
I think we can remove this test because the last one tested the retrieved result scenario and we had two results.